### PR TITLE
MBL-739: ActivitySampleFriendBackingViewHolder - ViewModel migrated to RXJava2

### PR DESCRIPTION
**Pull request by Arkariang**
# 📲 What

- One more migration

# 👀 See

https://github.com/kickstarter/android-oss/assets/4083656/ec4c27cc-3847-4c6b-beab-3d5f5104ce8d

|  |  |

# 📋 QA

- Press the "See all activity" once you've got the "Since you've been gone ..." card on Discovery

# Story 📖

[MBL-739](https://kickstarter.atlassian.net/browse/MBL-739)

[MBL-739]: https://kickstarter.atlassian.net/browse/MBL-739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt [hunk 1]
```diff
@@ -4,42 +4,45 @@ import android.text.Html
...
             .subscribe { activity ->
                 val user = activity?.user()
                 val project = activity?.project()
                 if (user != null && project != null) {
                     binding.activityTitle.visibility = View.GONE
                     user.avatar().small()?.let { url ->
-                        Picasso.get().load(url)
-                            .transform(CircleTransformation())
-                            .into(binding.activityImage)
+                        binding.activityImage.loadCircleImage(url)
```
Comment (ycheng-kickstarter) [inline app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt:43]: Curious for my own understanding why this change was required. Thanks!

File: app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt [hunk 2]
```diff
@@ -66,6 +70,11 @@ class ActivitySampleFriendBackingViewHolder(
         delegate.activitySampleFriendBackingViewHolderProjectClicked(this, activity.project())
     }
 
+    override fun destroy() {
+        this.vm.clearDisposables()
```
Comment (ycheng-kickstarter) [inline app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt:74]: Don't forget to clear the disposable in this viewholder too, not just the one in the viewmodel!

**Review by ycheng-kickstarter — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt (hunk 2 above)
Comment (Arkariang) [reply app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt:74]: done!

File: app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt (hunk 1 above)
Comment (ycheng-kickstarter) [reply app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt:43]: Got it, will keep a look out for other opportunities to do this

**Review by ycheng-kickstarter — APPROVED** [review]
